database/migrations.php

```diff
@@ -25,9 +25,16 @@ function migate(mysqli $db)
     // Create the tables we need
     $create_users_statement = "
     CREATE TABLE Users
-    (
-    id INTEGER AUTO_INCREMENT PRIMARY KEY,
-    username VARCHAR(40) NOT NULL
+    (User_ID INTEGER AUTO_INCREMENT PRIMARY KEY,
+    Username VARCHAR(30) NOT NULL,
+    PASSWORD VARCHAR(30) NOT NULL,
+    Email VARCHAR(30) NOT NULL,
+    FirstName VARCHAR(20) NOT NULL,
+    LastName VARCHAR(30) NOT NULL,
+    Address VARCHAR(100) NOT NULL,
+    User_Join_Time TIMESTAMP(0) NOT NULL DEFAULT CURRENT_TIMESTAMP(0),
+    IsBuyer BOOLEAN NOT NULL DEFAULT True, 
+    IsSeller BOOLEAN NOT NULL DEFAULT False 
```

Comment:
For what I understood (and how we are doing it everywhere else) attributes should be camelCase, and I think we should keep id as just id as whenever we reference it we will have users.id, but that is something we can discuss)

Reply:
Ok, that makes sense, I'll make the changes